workflow-templates/im-deploy-az-app-manually.yml

```diff
@@ -231,25 +233,34 @@ jobs:
       # ARM_SUBSCRIPTION_ID is an env-level secret
       # - name: Create a deployment slot
       #   run: |
-      #     az webapp deployment slot create \
+      #     az ${{ env.APP_TYPE }} deployment slot create \
```

Comment:
Would a name like `AZ_APP_TYPE` make it a little clearer what this env is for? I had to scroll all the way here before I knew for sure what it was being used to represent. The values were a clue, but I wasn't totally positive. This may just be me though.